setup.cfg

```diff
@@ -1,7 +1,9 @@
 [flake8]
 max-line-length = 88
 max-complexity = 18
-select = B,C,E,F,W,T4,B9,B950
+extend-ignore = E203,W503
```

Comment:
@mml5bg: Is the fix now, that Flake8 is ignoring what Black is changing? (I thought more, that we align code formatting configuration of both linters)

Reply:
Ah - ok, should have read the linked stuff first. :-"
So, it's fine